analysis/data_utils.py

```diff
@@ -100,18 +100,17 @@ def filter_max_time(experiment_df, max_time):
 
 # Creating "snapshots" (see README.md for definition).
 
-_DEFAULT_BENCHMARK_SAMPLE_NUM_THRESHOLD = 1.0
+_MIN_FRACTION_OF_ALIVE_TRIALS_AT_SNAPSHOT = 0.5
```

Comment:
I'm confused, where is this variable used?

Reply:
right below (just renamed this to be less confusing).

Reply:
ACK